numpy/core/tests/test_umath.py

```diff
@@ -1861,8 +1862,8 @@ class A(object):
             def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
                 return self, ufunc, method, inputs, kwargs
 
+        inner1d = ncu_tests.inner1d
```

Comment:
This change and the import on top are just so one can run
```
python3 numpy/core/tests/test_umath.py
```
in an environment where the current version of numpy is installed